apps/account-api/test/delegations.controller.e2e-spec.ts

```diff
@@ -119,6 +120,58 @@ describe('Delegation Controller', () => {
   describe('/v1/delegation/revokeDelegation', () => {
     let delegationObj: RevokeDelegationPayloadResponseDto;
 
+    it('(GET) /v1/delegation/revokeDelegation/:accountId/:providerId with invalid accountId', async () => {
+      const providerId = provider.msaId?.toString();
+      const { keypair } = users[1];
+      const invalidAccountId = `${keypair.address.slice(0, -1)}5`;
```

Comment:
What if the keypair already ends in '5'?
You could just generate a keypair here, guaranteed not to be an MSA. For instance:
```typescript
const invalidKeypair = new Keyring({ type: 'sr25519' }).createFromUri('//Alice//invalidUser');
```

Reply:
The point of this test is an invalid accountId, so a new keypair doesn't hit the code path.
Point taken about the 5, I added `5H` instead so it passes dto validation, but not `isValidAddress`

Reply:
`isValidAddress` can probably be moved to the DTO validation with decorators, but leaving it here for now. Also, there was a version that had an `ss58` validation decorator for the accountId, but I took it out, as the validation refactor is going to happen for v1.2.